Packs/PicusAutomation/Playbooks/Picus.yml

```
@@ -0,0 +1,888 @@
+    type: regular
+    task:
+      id: 35fd0482-f416-45fe-889f-7402a63b1764
+      version: -1
+      name: Get Secure Attacks
+      description: Get secure attack results
+      script: '|||picus-get-attack-results'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "15"
+    scriptarguments:
+      attacker_peer:
+        simple: ${Picus.param.attacker_peer}
+      days:
+        simple: ${Picus.param.days}
+      result:
+        simple: secure
+      victim_peer:
+        simple: ${Picus.param.victim_peer}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -330,
+          "y": 1620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: ed6d3b73-e847-4815-893b-520cb0a228e3
+    type: regular
+    task:
+      id: ed6d3b73-e847-4815-893b-520cb0a228e3
+      version: -1
+      name: Run Attacks
+      description: Run attacks on Picus based on previous task results
+      script: '|||picus-run-attacks'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "17"
+    scriptarguments:
+      attacker_peer:
+        simple: ${Picus.param.attacker_peer}
+      threat_ids:
+        simple: ${Picus.attackresults.results.threat_ids}
+      variant:
+        simple: ${Picus.param.variant}
+      victim_peer:
+        simple: ${Picus.param.victim_peer}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2260
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "5":
+    id: "5"
+    taskid: fac30ccd-3e8d-44bc-8349-470007a1e6a3
+    type: regular
+    task:
+      id: fac30ccd-3e8d-44bc-8349-470007a1e6a3
+      version: -1
+      name: Get Threat Results
+      description: Get threat results of played attacks
+      script: '|||picus-get-threat-results'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      attacker_peer:
+        simple: ${Picus.param.attacker_peer}
+      threat_ids:
+        simple: ${Picus.runattacks}
+      variant:
+        simple: ${Picus.param.variant}
+      victim_peer:
+        simple: ${Picus.param.victim_peer}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 3000
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "6":
+    id: "6"
+    taskid: a1afc992-9e65-49dc-8219-57ae2025c0b1
+    type: regular
+    task:
+      id: a1afc992-9e65-49dc-8219-57ae2025c0b1
+      version: -1
+      name: Filter Insecure Attacks
+      description: Filter insecure attacks
+      script: '|||filter-insecure-attacks'
```

Please verify that playbook works after the change. 
```suggestion
      script: '|||picus-filter-insecure-attacks'
```

---

I tested it works.